homeassistant/helpers/template.py

```diff
@@ -335,10 +335,10 @@ def _resolve_state(self, entity_id_or_state):
         return None
 
 
-def forgiving_round(value, precision=0):
+def forgiving_round(value, precision=0, method='common'):
     """Rounding filter that accepts strings."""
     try:
-        value = round(float(value), precision)
+        value = round(float(value), precision, method)
```

Comment:
@balloob That makes sense.. This was a shot in the dark.

Is there a way we can proxy to the jinja [do_round](https://github.com/pallets/jinja/blob/master/jinja2/filters.py#L657) filter?